e-commerce/shop-backend/package.json

```diff
@@ -3,16 +3,16 @@
   "version": "0.0.1",
   "scripts": {
     "build": "tsc",
-    "test": "npx knex migrate:rollback && npx knex migrate:up && npx knex seed:run && jest --detectOpenHandles",
-    "db:setup": "npx knex migrate:up && npx knex seed:run",
+    "test": "npx dbos rollback && npx dbos migrate && jest --detectOpenHandles",
+    "db:setup": "npx dbos migrate",
```

Comment:
Does migrate always seed?

Reply:
(That is to say, if you run with no preexisting database, are the products getting inserted into the store?

Reply:
Yes, this application is configured (in `dbos-config.yaml`) to always seed during migration.

Reply:
This is not described in any documentation that comes up in a search for "seed".